src/store/reducer.ts

```diff
@@ -6,13 +6,18 @@ import navReducer from './modules/nav/reducer';
 import tabsReducer from './modules/tabs/reducer';
 import searchBarReducer from './modules/searchBar/reducer';
 import headerReducer from './modules/header/reducer';
+import sensitiveReducer from './modules/sensitive/reducer';
+
 import farmerReducer from './modules/farmer/reducer';
+import dairyReducer from './modules/dairy/reducer';
 
 export default combineReducers<State>({
   drawer: drawerReducer,
   nav: navReducer,
   tabs: tabsReducer,
   searchBar: searchBarReducer,
   header: headerReducer,
+  sensitive: sensitiveReducer,
```

Comment:
maybe name this "sensitiveData"? or "sensitiveInfo"? so it is a noun like all the other ones